include/rcutils/logging.h

```diff
@@ -196,16 +221,14 @@ rcutils_logging_severity_level_from_string(
  * \param severity The severity level
  * \param name The name of the logger
  * \param timestamp The timestamp
- * \param format The format string
- * \param args The variable argument list
+ * \param log_str The string to be logged
  */
 typedef void (* rcutils_logging_output_handler_t)(
   const rcutils_log_location_t *,  // location
   int,  // severity
   const char *,  // name
   rcutils_time_point_value_t,  // timestamp
-  const char *,  // format
-  va_list *  // args
+  const char *  // log_str
```

Comment:
I switched to this for two reasons. 1) I wanted to make sure the log message was consistent for every output handler. 2) I wanted to reduce the amount of redundancy in both the runtime execution of formatting logs and in the code. From what I saw, there wasn't an easy way to provide named key replacement during log formatting in libraries such as log4cxx. This would require external libraries written on top of those to either reimplement the code that replaces those keywords or restructure/move the existing code in rcutils that does that formatting/replacement so that it could be used by the external libraries.